packages/fastnetmon/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: h0bbl3s <david.frisbee@gmail.com>
+
+pkgname="fastnetmon"
+pkgver=1.1.3.c74470e92
+pkgrel=1
+pkgdesc="ddos detection and network monitor using pf_ring"
+arch=(x86_64)
+url="www.fastnetmon.com"
+license=('GPL')
+groups=('blackarch' 'blackarch-networking')
+depends=('ndpi' 'log4cpp' 'libpfring' 'pfring-kernel' 'luajit' 'boost' 'boost-libs')
+makedepends=('pfring-kernel' 'libpfring' 'cmake' 'linux-headers')
+provides=('fastnetmon')
+source=("git+https://github.com/h0bbl3s/fastnetmon.git")
+md5sums=('SKIP')
+
+build() {
+	cd "${srcdir}/fastnetmon/src"
```

Comment:
Indentation is 2 spaces for all PKGBUILD.

Reply:
fixed this in all PKGBUILD files